### PR TITLE
[SCB-396] java-sdk needs to distinguish the services belongs to the devops stage(environment)

### PR DESCRIPTION
**Pull request by weichao666**
…evops stage(environment)

Follow this checklist to help us incorporate your contribution quickly and easily:

 - [x] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/SCB) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [x] Each commit in the pull request should have a meaningful subject line and body.
 - [x] Format the pull request title like `[SCB-XXX] Fixes bug in ApproximateQuantiles`, where you replace `SCB-XXX` with the appropriate JIRA issue.
 - [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [x] Run `mvn clean install` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---

### CODE REVIEW

**Comment by weichao666** [comment]
sdk support registers service environment according to environment variables.
servcie center registers the service interface, requesting the field with the identification phase in the body,{"service":{"serviceId":"xxx", "environment":"production"}}.
the name of the environment variable should be aligned with GOSDK and be unified.
the default value of this field is blank.

**Comment by weichao666** [comment]
image

File: service-registry/src/main/java/org/apache/servicecomb/serviceregistry/api/registry/MicroserviceFactory.java [hunk 1]
```diff
@@ -60,6 +62,7 @@ private Microservice createMicroserviceFromDefinition(Configuration configuratio
     microservice.setPaths(ConfigurePropertyUtils.getMicroservicePaths(configuration));
     Map<String, String> propertiesMap = MicroservicePropertiesLoader.INSTANCE.loadProperties(configuration);
     microservice.setProperties(propertiesMap);
+    microservice.setEnvironment(configuration.getString(SERVICECOMB_ENV, DEFAULT_SERVICECOMB_ENV));
```
Comment (wujimin) [inline service-registry/src/main/java/org/apache/servicecomb/serviceregistry/api/registry/MicroserviceFactory.java]: PR title said that "instance" stage, but IMPL is microservice
so what's the real purpose?

Comment (weichao666) [reply service-registry/src/main/java/org/apache/servicecomb/serviceregistry/api/registry/MicroserviceFactory.java]: The real purpose is microservice,the title in issue is too old, it's been modified now.

Comment (weichao666) [reply service-registry/src/main/java/org/apache/servicecomb/serviceregistry/api/registry/MicroserviceFactory.java]: image

Comment (weichao666) [reply service-registry/src/main/java/org/apache/servicecomb/serviceregistry/api/registry/MicroserviceFactory.java]: image

Comment (wujimin) [reply service-registry/src/main/java/org/apache/servicecomb/serviceregistry/api/registry/MicroserviceFactory.java]: ok

**Pull request merged**